analyses/plotAndPublish/index.php

```diff
@@ -36,9 +66,15 @@
     padding: 0 0.5em; 
 }
 </style>
+<script>
+function ClearFields() {
+
+     document.getElementById("match").value = "";
+}
+</script>
 </head>
 <body>
-<h1><?php echo getcwd(); ?></h1>
+<h1><?php echo substr(dirname(__FILE__), strpos(dirname(__FILE__), 'SliceTest/')); ?></h1>
```

Comment:
Remove full path from the top of the page. I think it is cleaner now (and somehow also safer)